packages/vitest-environment-nuxt/package.json

```
@@ -0,0 +1,64 @@
+{
+  "name": "vitest-environment-nuxt",
+  "version": "0.4.2",
+  "description": "An vitest environment with support for testing code that needs a Nuxt runtime environment.",
+  "repository": "danielroe/vitest-environment-nuxt",
+  "keywords": [
+    "nuxt",
+    "vitest",
+    "vitest-environment"
+  ],
+  "author": {
+    "name": "Daniel Roe <daniel@roe.dev>",
+    "url": "https://github.com/danielroe"
+  },
+  "license": "MIT",
+  "scripts": {
+    "build": "unbuild",
+    "prepublishOnly": "pnpm test run && pinst --disable",
+    "postpublish": "pinst --enable"
+  },
+  "main": "./dist/index.cjs",
+  "module": "./dist/index.mjs",
+  "exports": {
+    "./config": {
+      "require": "./dist/config.cjs",
+      "import": "./dist/config.mjs",
+      "types": "./dist/config.d.ts"
+    },
+    "./utils": {
+      "require": "./dist/utils.cjs",
+      "import": "./dist/utils.mjs",
+      "types": "./dist/utils.d.ts"
+    },
+    "./module": {
+      "require": "./dist/module.cjs",
+      "import": "./dist/module.mjs",
+      "types": "./dist/module.d.ts"
+    },
+    ".": {
+      "require": "./dist/index.cjs",
+      "import": "./dist/index.mjs",
+      "types": "./dist/index.d.ts"
+    }
```

Apparently this is the spec - not related to this PR but thought might be good to change while I was thinking of it.

```suggestion
    "./config": {
      "types": "./dist/config.d.ts",
      "require": "./dist/config.cjs",
      "import": "./dist/config.mjs"
    },
    "./utils": {
      "types": "./dist/utils.d.ts",
      "require": "./dist/utils.cjs",
      "import": "./dist/utils.mjs"
    },
    "./module": {
      "types": "./dist/module.d.ts",
      "require": "./dist/module.cjs",
      "import": "./dist/module.mjs"
    },
    ".": {
      "types": "./dist/index.d.ts",
      "require": "./dist/index.cjs",
      "import": "./dist/index.mjs"
    }
```